app/services/settings/general/upsert.rb

```diff
@@ -5,7 +5,15 @@ module Upsert
       TAG_PARAMS = %w[sidebar_tags suggested_tags].freeze
 
       def self.call(settings)
-        cleaned_params = clean_params(settings)
+        params_to_clean = settings.except(:logo)
```

Comment:
@julianna, @Ridhwana, unless I missed it, there doesn't appear to be a spec for the general settings upsert. There is only one for the `::Settings::Upsert`. That spec covers whatever settings you pass in, so I imagine we're good? We also already have tests for the `LogoUploader`

I will however add E2E tests.

Reply:
I'm not seeing a spec either, @nickytonline, and feel that with the E2E tests for this, we should be good, but I'm curious about what @Ridhwana thinks!

Reply:
I don't see a spec file. I'll double check if one needs to be added, and it to our pod board to tackle at some point.